lib/email/receiver.rb

```diff
@@ -861,6 +872,45 @@ def process_forwarded_email(destination, user)
       true
     end
 
+    def forwarded_email_quote_forwarded(destination, user)
+      embedded = embedded_email_raw
+      raw = <<~EOF
+        #{@before_embedded}
+
+        [quote="#{I18n.t('emails.forwarded_message')}"]
+        #{PlainTextToMarkdown.new(embedded).to_markdown}
+        [/quote]
+        EOF
```

Comment:
We usually indent the end of the squiggly heredoc comment at the same level as the variable

```suggestion
      EOF
```